doc/openapi.yaml

```diff
@@ -0,0 +1,102 @@
+openapi: 3.0.3
+info:
+  title: Events API
+  version: 1.0.0
+servers:
+  - url: http://localhost:4000
+paths:
+  /api/events:
+    get:
+      summary: Get list of events
+      responses:
+        '200':
+          description: List of events
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EventsResponse'
+    post:
+      summary: Create a new event
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/EventInput'
+      responses:
+        '201':
+          description: Event created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Event'
+  /api/events/{id}:
+    get:
+      summary: Get event by ID
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: integer
+      responses:
+        '200':
+          description: Event detail
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Event'
+        '404':
+          description: Event not found
```

Comment:
tady by bylo take dobre uvest 
```
content:
    application/json:
       schema:
       $ref: '...'
```

a dat tam schema, ktere odpovida tve chybe ze serveru:
```
        return res.status(404).json({ error: "Event not found" });
```

to same u chybove odpovedi na create event